example-playbooks/kernel_version_enforcer/ubuntu/grub.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Procedure below doesn't seem to be standard.
Here is an example of a procedure I'm aware of: https://askubuntu.com/questions/100232/how-do-i-change-the-grub-boot-order.

In gist you need to change `GRUB_DEFAULT` value in `/etc/default/grub` and then run `sudo update-grub`.